nmrfx-processor/src/main/java/org/nmrfx/processor/optimization/PeakMatcher.java

```diff
@@ -292,9 +292,8 @@ public double globalScore(boolean dump) {
     }
 
     public void processPPMFile(String fileName) {
-        File file = new File(fileName);
-        try {
-            Scanner in = new Scanner(file);
+        try (Scanner in = new Scanner(new File(fileName))) {
```

Comment:
This is a third way to parse a file, which is more recent that BufferedReader but older than Files.lines().
It's still valuable if the code inside the stream must throw a checked exception.

Reply:
replaced with Files.lines()